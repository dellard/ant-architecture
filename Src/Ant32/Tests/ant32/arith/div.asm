#@tests div

	lc	r3, 5
	lc	r4, 10

	div	r5, r4, r3
	div	r6, r3, r5		#fractional
	div	r0, r3, r0		#div-by-0 exception
