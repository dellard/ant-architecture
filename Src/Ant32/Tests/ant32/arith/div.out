CPU ERROR: An exception occured while exceptions were disabled.
e0 = 0x00000000  e1 = 0x00000000  e2 = 0x80000010  e3 = 0x00000071
     (Shadow PC)      (INT mask)       (TLB addr)       (Exception)

ERROR: Exception occured with exceptions disabled.
